README.md

```diff
@@ -15,6 +15,14 @@ These need to be set when your final appsody container is deployed.
 need to set this to the name of the cert.pem that you've added to the root of
 this project. (example: QUAY_CA_FILE=ExampleRootCA.pem)
 
+### How it Works
+The application will forward your request to your `QUAY_HOST` environment
+variable i.e.
+https://your.initial.url/webhooks/push/trigger/id?token=token
+will forward to:
+https://your.quay.url/webhooks/push/trigger/id?token=token
```

Comment:
@SWheeler17 Could you kindly review this. We're converting the token query string to a basic auth format:
```
https://$token:token@your.quay.url/webhooks/push/trigger/id
```
^ Please double check